tests/unit/test_items_jsonschema.py

```diff
@@ -98,3 +100,26 @@ def test_new_acquisition(item_schema, item_lib_martigny_data_tmp):
         acq_date = datetime.date.today().strftime('%Y/%m/%d')
         item_lib_martigny_data_tmp['acquisition_date'] = acq_date
         validate(item_lib_martigny_data_tmp, item_schema)
+
+
+def test_temporary_item_type(item_schema, item_lib_martigny):
+    """Test temporary item type for item jsonschemas."""
+    data = item_lib_martigny
+
+    # tmp_itty cannot be the same than main_itty
```

Comment:
Need the other test, when librarian changes `item_type` to equal the `temp_item_type`

Reply:
It will raised the same error. No need write another test.